CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
 ## Version 2.2.0-beta3
+- DomainNameRoleInstanceTelemetryInitializer is obsolete.
+- New AzureWebAppRoleEnvironmentTelemetryInitializer telemetry initializer that populates role name and role instance name for Azure Web Apps.
```

Comment:
Use single quote mark to make AzureWebAppRoleEnvironmentTelemetryInitializer look like this: `AzureWebAppRoleEnvironmentTelemetryInitializer`